README.md

```diff
@@ -4,15 +4,40 @@
 
 You've been contracted by Flatiron Bank to consult on their online banking application. Specifically, they've asked you to build out their user login process. Based on your knowledge of sessions, build out the log in and log out features, and ensure that users that log in can only see their own balance.
 
-## Notes
-+ We're not using a database - instead, when you spin up the application using Shotgun, we've created a few sample users. Take a minute to look at the User model and the sample users in `User.rb`.
+## Helper Methods
+
+MVC architecture relies heavily on the principle of separation of concerns. We make sure that we have a different model for every class we build, that we only have one erb file per view, etc. This even extends to the purposes each of these files has. A model handles our Ruby logic, our controllers handle the HTTP requests and connect to our models, and our views either take in or display data to our users.
+
+This means that we want to minimize the amount of logic our views contain. Our views should never directly pull from the database (ie. `User.all`, etc). All of that should be taken care of in the controller actions, and the data should be passed to the view via a specific controller action.
+
+But if you think about most web applications you use, there is information on most pages that are dependent on being logged in. You can see a lot information if you are logged in, and practically none if you're not. So how can you handle that sort of application flow without logic?
```

Comment:
I would expand on this just a bit so that students don't think helper methods are for handing logged/not logged in state. Helpers make it cleaner to add logic to our views